pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -1,7 +1,9 @@
-{ stdenv, lib, fetchurl, jdk, zip, unzip, bash, writeScriptBin, coreutils, makeWrapper, which, python
+{ stdenv, lib, fetchurl, jdk, zip, unzip, bash, writeCBin, coreutils, makeWrapper, which, python
 # Always assume all markers valid (don't redownload dependencies).
 # Also, don't clean up environment variables.
 , enableNixHacks ? false
+# Apple dependencies
+, libcxx, CoreFoundation, CoreServices, Foundation
```

Comment:
We want the current version of `CoreFoundation` even though we have 10.10 `CoreFoundation` already because Bazel uses `CFURL` APIs that only exist in the newer versions:
https://github.com/bazelbuild/bazel/blob/8d782751aa34a1fbb8c3ad38047bb7de2baf1bdb/src/main/cpp/blaze_util_darwin.cc#L106
https://developer.apple.com/documentation/corefoundation/1542764-cfurlcopyresourcepropertyforkey